src/main/kotlin/blackjack/domain/card/state/rule/Hit.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.domain.card.state.rule
+
+import blackjack.domain.card.PlayingCard
+import blackjack.domain.card.PlayingCards
+import blackjack.domain.card.state.Running
+import blackjack.domain.card.state.State
+
+class Hit(override val cards: PlayingCards) : Running(cards) {
+    override fun draw(playingCard: PlayingCard): State {
+        cards.add(playingCard)
```

Comment:
이미 상위 Running class에서 cards: PlayingCards를 오버라이드 하고 있고, 멤버변수로 가지고있기 때문에 하위에서 한 번 더 오버라이드할 필요는 없어요.
생성자에서는 파라미터로만 받고, 부모클래스에 잘 넘겨주기만 하면, 상위 부모 클래스의 멤버변수인 cards를 접근할 수 있게 됩니다 :)

```suggestion
class Hit(cards: PlayingCards) : Running(cards) {
    override fun draw(playingCard: PlayingCard): State {
        cards.add(playingCard)
```

Reply:
작은 부분 까지 봐주셔서 감사합니다 😄 추후에는 이런 실수가 없도록 신경 쓰도록 하겠습니다 🙏 
아래와 같이 수정하였습니다!

- https://github.com/next-step/kotlin-blackjack/commit/3afa4072e7336154407046f41c10b804f754b7d4